rundeckapp/build.gradle

```diff
@@ -294,12 +294,8 @@ bootRun {
 }
 
 tasks.withType(GroovyCompile) {
-    def additionalSpecDir = new File(project.buildDir, 'openapi')
-    doFirst {
-        additionalSpecDir.mkdirs()
-    }
```

Comment:
we use this to merge openapi specs with Rundeck Enterprise based on Rundeck core, so this change in this section should be reverted